content/webapp/components/ItemRequestModal/RequestDialog.tsx

```diff
@@ -86,6 +87,15 @@ const RequestDialog: FunctionComponent<RequestDialogProps> = ({
     item.availableDates && dateAsValue(new Date(item.availableDates[0].from))
   );
 
+  // we know an item is offsite/deepstore if its 1st available date is more than 3 days in the future
+  const isOffsiteDeepstoreItem =
+    item.availableDates &&
+    new Date(item.availableDates[0].from) > addDays(today(), 3);
+
+  const pickupDeadline = isOffsiteDeepstoreItem
+    ? 'Item requests for offsite material need to be placed by 10am, 10 working days before your visit'
```

Comment:
Instead of a boolean, what about indicating the lead time in the message whatever the source so we are not making the slightly obfuscated connection to Deepstore / offsite items? We also know the time so can indicate the exact lead time when the message is displayed.

For example:
 
"The earliest available date to view this item is in 3 days on the 12/07/2024 if you order before 10am.

Reply:
Wondering if you could move (whatever) this copy (end ups being) to [microcopy](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/data/microcopy.tsx) and the use it both here and in the tests?

The goal of microcopy was to have as much of it in one place so non-devs would feel comfortable making small edits. I don't think it's actually used like that, but if we move to a different solution, at least most of it is easily findable there!

Reply:
I don't like this boolean much either but we decided to have DeepStore/offsite items' location be "closed stores" on the work page, and it sounds like CI wants to give some context as to why some items take longer than others. 
See https://wellcome.slack.com/archives/C05JE6PE5PG/p1718200849717969
Maybe we can revisit this when they update the Terms of use to be as seen on this [slide](https://wellcomecloud.sharepoint.com/:p:/r/sites/wc2/cr/cm/Access%20Management/Manual%20Requests%20project%202023/Manual%20request%20process%20examples.pptx?d=w0dfe05476d7b484dbac4b868afba28c7&csf=1&web=1&e=6R23IV&nav=eyJzSWQiOjI2OSwiY0lkIjo0MDU0ODUxMTM3fQ)

Reply:
Decided at review meeting to go with Robert's suggestion, minus the 10am bit as the date displayed already takes into account what time of day the request is made